src/workloads/execution/MultiDeletesBatchedWithSecondaryIndexes_16KB.yml

```diff
@@ -0,0 +1,39 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+Description: >
+  Deletes a range of documents using the BATCHED_DELETE query exec stage while performing writes on another collection.
+  Secondary indexes are created to see how they might affect performance. Documents are approximately of size ~1KB.
+
+Keywords:
+- RunCommand
+- Loader
+- LoggingActor
+- CrudActor
+- insert
+- delete
+- batch
+- deleteMany
+- latency
+- secondary indexes
+
+# TODO move the Clients section to src/phases/execution/MultiDeletes.yml once TIG-3950 is resolved.
+Clients:
+  Default:
+    QueryOptions:
+      # Allow for longer duration since the loading of the data may take a while.
+      socketTimeoutMS: 3_600_000  # = 1 hour
+
+LoadConfig:
+  Path: "../../phases/execution/MultiDeletes.yml"
+  Parameters:
+    TargetDocumentSizeBytes: 16_384  # 16KB
+    DocumentCount: 62_500
```

Comment:
As we discovered offline, we're probably missing a 0 in the document count for the 16kB workloads. In general, let's make sure that mass deletions last at least a couple of minutes.

Reply:
Document counts have been modified. Also changed the Loader to be multithreaded to speed up the workload setup. With 1_000_000 docs for 16kb I saw some instances where the duration was under 2 minutes, so I bumped it to 1_500_000.

Patch: https://spruce.mongodb.com/version/626c009430661566a401e498/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC